source/2019-05-20-ember-2019-roadmap-call-for-posts.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Ember's 2019 Roadmap: Call for Blog Posts"
+author: Katie Gengler
+tags: Recent Posts, Roadmap, 2019
+responsive: true
+---
+
+Ember in 2018 (and 2019) has been more exciting than ever! We've been working diligently to advance the project based on our [2018 Roadmap](https://github.com/emberjs/rfcs/blob/master/text/0364-roadmap-2018.md), but while we work on that, it is time to think about the future. 
```

Comment:
Ember in 2018 (and 2019) has been more exciting than ever! We've been working diligently to advance the project based on our [2018 Roadmap](https://github.com/emberjs/rfcs/blob/master/text/0364-roadmap-2018.md), but while we work to finish that, it is time to think about the future.

Reply:
```suggestion
It's time to think about the future of Ember, and we need your ideas.

The core teams and community have been working diligently to advance the project based on our [2018 Roadmap](https://github.com/emberjs/rfcs/blob/master/text/0364-roadmap-2018.md). In all of Ember's history, 2018 and 2019 were more exciting than ever, and we'd like to know where you think that momentum could take us.
```